src/mlcompute.cs

```diff
@@ -2121,4 +2207,73 @@ interface MLCInferenceGraph {
 		[Export ("executeWithInputsData:lossLabelsData:lossLabelWeightsData:outputsData:batchSize:options:completionHandler:")]
 		bool Execute (NSDictionary<NSString, MLCTensorData> inputsData, [NullAllowed] NSDictionary<NSString, MLCTensorData> lossLabelsData, [NullAllowed] NSDictionary<NSString, MLCTensorData> lossLabelWeightsData, [NullAllowed] NSDictionary<NSString, MLCTensorData> outputsData, nuint batchSize, MLCExecutionOptions options, [NullAllowed] MLCGraphCompletionHandler completionHandler);
 	}
+
+	[TV (14,5)][Mac (11,3)][iOS (14,5)]
+	enum MLCComparisonOperation {
+		Equal = 0,
+		NotEqual = 1,
+		Less = 2,
+		Greater = 3,
+		LessOrEqual = 4,
+		GreaterOrEqual = 5,
+		LogicalAND = 6,
+		LogicalOR = 7,
+		LogicalNOT = 8,
+		LogicalNAND = 9,
+		LogicalNOR = 10,
+		LogicalXOR = 11,
```

Comment:
`AND` or `And`? TBH my head says uppercase but guidelines says otherwise :P

Reply:
AND is not and acronym, NADN and NOR.. maybe